### PR TITLE
[Makeen] [Bug] trailing slash in user domain

### PR DESCRIPTION
**Pull request by its-aazizi**
- Updated main-cf-template to not have protocol and trailing slash in the stack output.
- Handled protocol and trailing slash on client side just to be safe.

### CODE REVIEW

File: src/utils/getDomainName.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const getDomainName = (url: string) =>
+	url?.startsWith("http://") || url.startsWith("https://") ? url.split("//")[1].replace("/", "") : url;
```
Comment (vinguthh) [inline src/utils/getDomainName.ts:2]: it should be safe to remove the last trailing slash. Something like this:
```
if (url.endsWith('/')) {
        return url.slice(0, -1);
    }
    return url;
```

Comment (its-aazizi) [reply src/utils/getDomainName.ts:2]: @vinguthh 
That's what this is doing 
`url.split("//")[1].replace("/", "")`

Comment (its-aazizi) [reply src/utils/getDomainName.ts]: @vinguthh updated the condition, found an edge case. Resolved

Comment (vinguthh) [reply src/utils/getDomainName.ts]: But this assume that the url in this type of format: `http://something/`, then we will get `http://something`. If the url is `http://something/somethingelse`, then we will get `http://somethingsomethingelse`. I found it's safe to not assume the format of url but just to remove the last trailing slash.

**Pull request merged**